cli.py

```
@@ -1,8 +1,185 @@
+                if args.verbose and status_info.get("last_error"):
+                    print(f"       Last Error: {status_info['last_error']}")
+
+        # Mission Phase Section
+        print("\n🚀 MISSION PHASE")
+        print("-" * 70)
+        try:
+            state_machine = StateMachine()
+            current_phase = state_machine.current_phase
+            print(f"  Current Phase: {current_phase.value}")
+            print(f"  Description:   {_get_phase_description(current_phase.value)}")
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)